### YamlMime:PythonClass
uid: azure.ai.ml.exceptions.InvalidVSCodeRequestError
name: InvalidVSCodeRequestError
fullName: azure.ai.ml.exceptions.InvalidVSCodeRequestError
module: azure.ai.ml.exceptions
inheritances:
- azure.ai.ml.exceptions.MlException
summary: 'Exception raised when VS Code Debug is invoked with a remote endpoint.

  VSCode debug is only supported for local endpoints.'
constructor:
  syntax: InvalidVSCodeRequestError(error_category='UserError', msg=None)
  parameters:
  - name: error_category
    defaultValue: UserError
  - name: msg
    defaultValue: None
